src/util/content/ensureInlineContentArray.ts

```diff
@@ -6,13 +6,16 @@ import { TxtCodec } from '../../codecs/txt'
  * from within `BlockContent` if necessary.
  */
 export const ensureInlineContentArray = (
-  nodes: schema.Node[]
+  nodes: schema.Node | schema.Node[]
 ): schema.InlineContent[] => {
-  return nodes.reduce((prev: schema.InlineContent[], node) => {
-    return schema.isInlineContent(node)
-      ? [...prev, node]
-      : 'content' in node
-      ? [...prev, ...node.content]
-      : [...prev, TxtCodec.stringify(node)]
-  }, [])
+  return (Array.isArray(nodes) ? nodes : [nodes]).reduce(
+    (prev: schema.InlineContent[], node) => {
+      return schema.isInlineContent(node)
+        ? [...prev, node]
+        : 'content' in node
```

Comment:
I don't think this check is enough. It'll be a false positive for things like [`QuoteBlock`s](https://github.com/stencila/schema/blob/e88891e22b1028e5c6610c5a60d18a298fb604e0/schema/QuoteBlock.schema.yaml#L17-L22) which have `BlockContent` arrays for `content`.

Reply:
Good catch!. Fixed and added tests.